quickstarts/karaf-camel-amq-template.json

```diff
@@ -85,21 +85,21 @@
       "from": "[a-zA-Z0-9]{40}"
     },
     {
-      "name": "ACTIVEMQ_SERVICE_NAME",
-      "displayName": "ActiveMQ Broker Service",
+      "name": "AMQ_SERVICE_NAME",
```

Comment:
If you look further down in the template, you'll see that these parameters are referenced on the container spec to configure environment variables. So they will need updating as well.

The naming also needs to be aligned with what the quickstart application is expecting. See the `ConnectionFactory` configuration here:

https://github.com/fabric8-quickstarts/karaf-camel-amq/blob/fuse-7.1.x.redhat/src/main/resources/OSGI-INF/blueprint/camel-context.xml#L37-L42

`AMQP_PARAMETERS` is missing so we should probably add that to the template.